superagi/agent/agent_prompt_builder.py

```diff
@@ -122,13 +122,13 @@ def replace_task_based_variables(cls, super_agi_prompt: str, current_task: str,
                 completed_tasks_arr.append(task['task'])
             super_agi_prompt = super_agi_prompt.replace("{completed_tasks}", str(completed_tasks_arr))
 
-        base_token_limit = TokenCounter.count_message_tokens([{"role": "user", "content": super_agi_prompt}])
+        base_token_limit = TokenCounter().count_message_tokens([{"role": "user", "content": super_agi_prompt}])
         pending_tokens = token_limit - base_token_limit
         final_output = ""
         if "{task_history}" in super_agi_prompt:
             for task in reversed(completed_tasks[-10:]):
                 final_output = f"Task: {task['task']}\nResult: {task['response']}\n" + final_output
-                token_count = TokenCounter.count_message_tokens([{"role": "user", "content": final_output}])
+                token_count = TokenCounter().count_message_tokens([{"role": "user", "content": final_output}])
```

Comment:
make it class method if there is no need of any constructor params.

Reply:
done sir made the rest of the functions as static